src/Http/Http/src/Features/FormFeature.cs

```diff
@@ -87,6 +99,11 @@ public bool HasFormContentType
                 return true;
             }
 
+            if (_request is null)
+            {
+                return false;
+            }
+
             var contentType = ContentType;
```

Comment:
Personally, I think it would be clearer to check whether `contentType` is null (rather than testing `_request`) - the fact that a null `_request` yields a null `ContentType` seems like an implementation detail.